server/routes/varyingLicences/handlers/spoDiscussion.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express'
+import LicenceService from '../../../services/licenceService'
+
+export default class SpoDiscussionRoutes {
+  constructor(private readonly licenceService: LicenceService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    res.render('pages/vary/spoDiscussion')
+  }
```

Comment:
Are the answers to these questions ever prepopulated? Or just collected the once and never redirected back to this page with a value set? I don't see values being passed in here, that's all.

Reply:
Ah - probably set on the licence itself!

Reply:
The answers do get pre-populated from the licence. The licence object is already in the view context as default since its in `res.locals`